### PR TITLE
[New] Add smart-app-banner w/ npm auto-update

### PR DESCRIPTION
**Pull request by ralphchung**
Pull request for issue: #12488

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/kudago/smart-app-banner
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/smart-app-banner
 * License and its reference: [MIT](https://github.com/kudago/smart-app-banner/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 19
   - Count of stars: 350
   - Count of forks: 176
 * NPM download stats (optional):
   - Downloads in the last day: 29
   - Downloads in the last week: 3,010
   - Downloads in the last month: 11,021

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/smart-app-banner/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "type": "git",
+    "url": "git://github.com/kudago/smart-app-banner.git"
+  },
+  "keywords": [
+    "smart",
+    "banner",
+    "smartbanner",
+    "android",
+    "iphone",
+    "application"
```
Comment (sufuf3) [inline ajax/libs/smart-app-banner/package.json]: Let's add more keywords: https://github.com/kudago/smart-app-banner/blob/master/bower.json#L17-L20
Many thanks

File: ajax/libs/smart-app-banner/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "smart-app-banner",
+  "filename": "smart-app-banner.js",
+  "description": "Smart banner for Android or iOS",
```
Comment (sufuf3) [inline ajax/libs/smart-app-banner/package.json]: Let's use this one as the description: https://github.com/kudago/smart-app-banner/blob/master/bower.json#L8 which is the same with README.
Thank you

**Comment by ralphchung** [comment]
@sufuf3 I've tried to follow your instructions, but I am not sure whether what I did was perfectly correct.
Please check it.

Thank you very much!

**Comment by ralphchung** [comment]
@sufuf3 I've tried to follow your instructions but am not sure whether what I did was perfectly correct.

Please check it.

Thank you very much!

**Comment by sashberd** [comment]
@sufuf3 The PR is waiting FYR

**Comment by sashberd** [comment]
Moved to #13739

**Pull request closed without merging**